src/ILCompiler.WebAssembly/src/CodeGen/ILToWebAssemblyImporter.cs

```diff
@@ -1869,6 +1869,21 @@ private void ImportCall(ILOpcode opcode, int token)
                 }
             }
 
+            // Hard coded InternalCall mappings for mono interoperability
+            if (callee.IsInternalCall)
+            {
+                var metadataType = callee.OwningType as MetadataType;
+                // See https://github.com/dotnet/runtime/blob/9ba9a300a08170c8170ea52981810f41fad68cf0/src/mono/wasm/runtime/driver.c#L400-L407
+                // Mono have these InternalCall methods in different namespaces but just mapping them to CoreRT.WebAssembly.MonoBridge.
+                if (metadataType != null && (metadataType.Namespace == "WebAssembly.JSInterop" && metadataType.Name == "InternalCalls" || metadataType.Namespace == "WebAssembly" && metadataType.Name == "Runtime"))
+                {
+                    var coreRtJsInternalCallsType = _compilation.TypeSystemContext
+                        .GetModuleForSimpleName("CoreRT.WebAssembly.MonoBridge")
```

Comment:
Sorry - didn't notice this earlier. We have a convention here that these kinds of libraries are called "System.Private.*".

This is hardcoded in some places, like here:

https://github.com/dotnet/corert/blob/9fd573816ac81719f32f4b5635896a651a5c91ce/src/dir.props#L10-L13

This makes sure they get properly packaged and binplaced for the `dotnet` integration.

Can you name this to something like `System.Private.WebAssembly`?

Reply:
Sure, np